plugins/070.py

```diff
@@ -0,0 +1,17 @@
+import requests,json,HTMLParser,sys,getopt
+
+from will.plugin import WillPlugin
+from will.decorators import respond_to
+
+class NulZeventigPlugin(WillPlugin):
+    @respond_to("070")
+    def nul_zeventig_plugin(self, message, argv):
+		if len(argv) > 1:
+			if len(sys.argv[1]) > 0:
+				r = requests.get("http://looq.nl/070/index.php?text="+"{}".format("+".join(sys.argv)))
```

Comment:
This is pretty bad... use requests like a man and provide the querystring key/values using a dict.